lib/place_v7.ml

```diff
@@ -457,8 +455,7 @@ let print_html_places_surnames_short conf _base _link_to_ind
             len
       | (pl, _ipl) :: list ->
           let str =
-            if very && List.length pl > 0 then List.nth pl 0
-            else places_to_string true pl
+            match pl with pl :: _ -> pl | _ -> places_to_string true pl
```

Comment:
there was a `if very &&  ...` here, is it okay to remove the condition on `very`?

Reply:
good catch!!